pkg/helpers/scheme/scheme.go

```diff
@@ -16,6 +11,10 @@ import (
 	securityv1 "github.com/openshift/api/security/v1"
 	"github.com/openshift/api/template"
 	"github.com/openshift/api/user"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
Worth moving this back where it was to undo the no-op changes to this file

Reply:
Thank you for review. Nice catch. Updated.